module-2/backend/week_two.md

```diff
@@ -4,38 +4,89 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. At a high level, what is ActiveRecord? What does it do/allow you to do?
-2. What is a migration?
-3. How does a table relate to a model?
-4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
-5. What do they allow you to do?
-6. What's the difference between agile workflow and waterfall method?
-7. What is the difference between `#new` and `#create`?
-8. At a basic level, what does cURL allow you to do?
-9. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
-10. Define foreign key, primary key, and schema.
-11. Describe the relationship between a foreign key on one table and a primary key on another table.
-12. What are the parts of an HTTP response?
-13. `Rack::Test` allows us to test our controllers in isolation. What are some of the methods it gives us to simulate the request/response cycle?
-14. Describe some techniques to make our Sinatra views more DRY. Give an example of when you would use these techniques.
+1. At a high level, what is ActiveRecord? What does it do/allow you to do?  
+ActiveRecord is an ORM for using Ruby with relational SQL databases. 
+ActiveRecord is a collection of methods written to streamline common Ruby methods. 
+
+2. What is a migration?  
+A migration is the instruction for how to alter/create tables in a database.   
+
+3. How does a table relate to a model?  
+A model represents a table in a database. You can build methods on a model to interact with a table.   
+
+4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.  
+They are class methods. Class and Teacher would have a one to many relationship, a class has one teacher and teachers have many classes. Students and Class would have a many to many relationships as a class has many students and students have many classes.   
```

Comment:
I think we were trying to point out that you would need to have a specific instance of Class before you could use the methods that `belongs_to` offers you. For example:

```ruby
# Assuming our Class class has a `belongs_to :teacher` line
class_1704 = Class.find(1) # finds the specific class with an id of 1
mr_fritz      = class_1704.teacher # uses the teacher method that we now have to return a specific teacher associated with `class_1704`
```